### PR TITLE
Update Build Controller to v0.6.0

### PR DESCRIPTION
**Pull request by adambkaplan**
# Changes

- Update release manifest to use Shipwright Build v0.6.0.
- Move the namespace and TektonConfig RBAC definitions to the
  kubebuilder annotations.
- Added RBAC to manage the shipwright-build-controller deployment.
- Added RBAC manage the shipwright-build-controller service account.
- Added RBAC to create the namespaced role/rolebindings for the build
  controller

/kind feature

# Submitter Checklist

- [x] Includes tests if functionality changed/was added
- [ ] Includes docs if changes are user-facing
- [x] [Set a kind label on this PR](https://prow.k8s.io/command-help#kind)  
- [x] Release notes block has been filled in, or marked NONE

See [the contributor guide](https://github.com/shipwright-io/build/blob/master/CONTRIBUTING.md)
for details on coding conventions, github and prow interactions, and the code review process.

# Release Notes

```release-note
Upgrade the deployed build controller to v0.6.0
```

### CODE REVIEW

File: bundle/manifests/shipwright-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -212,88 +212,168 @@ spec:
...
+          resources:
+          - buildruns/finalizers
+          verbs:
+          - update
+        - apiGroups:
+          - shipwright.io
+          resources:
+          - buildruns/status
+          verbs:
+          - update
```
Comment (SaschaSchwarze0) [inline bundle/manifests/shipwright-operator.clusterserviceversion.yaml:235]: This could be merged into a single entry.

**Review by SaschaSchwarze0 — APPROVED** [review]
/lgtm

File: bundle/manifests/shipwright-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (adambkaplan) [reply bundle/manifests/shipwright-operator.clusterserviceversion.yaml:235]: Everything under `/bundle` is auto-generated by operator-sdk, and will be used in the listing on operatorhub.io.

**Comment by adambkaplan** [comment]
/approve

Self-approving

**Pull request merged**